### PR TITLE
Issue 757 rate capability

### LINKED ISSUE #757: Rate capability plots
**Summary**
Create a script to simulate rate capability experiments (how much capacity can be pulled out before we hit a given limit voltage at CC for different C-rates, see picture below).

image

 **Motivation**
This is a standard experiment on batteries and it would be pretty easy to reproduce in PyBaMM with what is available so far.

**Additional context**

### PR DESCRIPTION
**Pull request by ferranbrosa**
# Description

I have added an example script that generates rate capability plots. The implementation is not the best, so feedback on how to make it more efficient is welcome. In any case, I think it is a good first step to include in v0.2. 

Because it is an example, rather than a new feature, I haven't added to CHANGELOG.md.

Fixes #757 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/master/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ flake8`
- [x] All tests pass: `$ python run-tests.py --unit`
- [x] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [X] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: examples/scripts/DFN_high_Crate.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+var = pybamm.standard_spatial_vars
+var_pts = {var.x_n: 30, var.x_s: 30, var.x_p: 30, var.r_n: 10, var.r_p: 10}
+mesh = pybamm.Mesh(geometry, model.default_submesh_types, var_pts)
+
+# discretise model
+disc = pybamm.Discretisation(mesh, model.default_spatial_methods)
+disc.process_model(model)
+
+# solve model
+t_eval = np.linspace(0, 0.2, 1E4)
```
Comment (tinosulzer) [inline examples/scripts/DFN_high_Crate.py]: needs to be updated with t_eval in seconds

File: examples/scripts/rate_capability.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+plt.figure(1)
+plt.scatter(C_rates, capacities)
+plt.xlabel('C-rate')
+plt.ylabel('Capacity [Ah]')
+
+plt.figure(2)
+plt.scatter(currents * voltage_av, capacities * voltage_av)
+plt.xlabel('Power [W]')
+plt.ylabel('Energy [Wh]')
+
```
Comment (tinosulzer) [inline examples/scripts/rate_capability.py]: nice!

File: .gitignore [hunk 3]
```diff
@@ -91,3 +91,6 @@ cmake_install.cmake
 
 
 third-party/pybind11
+
+# LGM50 results
+results/LGM50/
```
Comment (tinosulzer) [inline .gitignore]: this can be removed?

File: examples/scripts/DFN_high_Crate.py (hunk 1 above)
Comment (ferranbrosa) [reply examples/scripts/DFN_high_Crate.py]: I did this such a long time ago that I thought someone else had done it haha. I have removed it now.

File: .gitignore (hunk 3 above)
Comment (ferranbrosa) [reply .gitignore]: Done

**Pull request merged**